blueprints/ember-redux/files/__root__/reducers/index.js

```diff
@@ -5,7 +5,7 @@ import { combineReducers } from 'npm:redux';
  * one or more other reducers which take on responsibility for
  * a discrete part of the state tree.
  *
- * Use the "ember generate reducer" command to create additional
+ * Use the "ember generate reducer [name]" command to create additional
```

Comment:
Ideally when you use the `ember g reducer xyz` or `ember d reducer xyz` it would automatically populate the `index.js` reducer in the same way that `ember g route` does today.